app/src/main/java/org/keepgoeat/presentation/my/AchievedGoalAdapter.kt

```diff
@@ -17,10 +22,22 @@ class AchievedGoalAdapter :
     ) {
     private lateinit var inflater: LayoutInflater
 
-    class AchievedGoalViewHolder(private val binding: ItemAchievedGoalBinding) :
+    interface updateGoalIdListener {
+        fun updateGoalId(data: AchievedGoal)
+    }
+
+    inner class AchievedGoalViewHolder(private val binding: ItemAchievedGoalBinding) :
```

Comment:
```suggestion
    class AchievedGoalViewHolder(private val binding: ItemAchievedGoalBinding) :
```
inner class는 정말 필요하지 않은 이상 사용하지 않는 것이 좋습니다! 아래 링크를 참고해보시면 좋을 것 같습니다~ onBind에서 필요한 context와 같이 파라미터가 추가로 더 필요하다면 data넘기듯이 파라미터로 전부 넘기면 되겠습니다~
[Kotlin과 Java의 Nested and Inner Classes를 알아보고, Nested classes를 왜 사용해야 하는지 알아본다.](https://thdev.tech/kotlin/2020/11/17/kotlin_effective_11/)